Makefile

```diff
@@ -755,7 +755,7 @@ format:
 
 GIT_TARGET_BRANCH ?= main
 check-format: format
-	git diff --name-only HEAD origin/$(GIT_TARGET_BRANCH) -- "*.go" | xargs --no-run-if-empty git diff --exit-code -- \
+	find . $(DONT_FIND) -name operator -prune -o -type f -name "*.go" | xargs --no-run-if-empty git diff --exit-code -- \
```

Comment:
This excludes the operator files but not the folder :see_no_evil: So the git diff runs on `./operator`. You need to exclude it somehow.